lib/xnvme_be_vfio_dev.c

```diff
@@ -179,8 +177,8 @@ xnvme_be_vfio_dev_open(struct xnvme_dev *dev)
 	ctrl = _vfio_cref_lookup(&dev->ident);
 	if (!ctrl) {
 		struct nvme_ctrl_opts ctrl_opts = {
-			.nsqr = nqueues,
-			.ncqr = nqueues,
+			.nsqr = XNVME_BE_VFIO_NQUEUES_MAX - 1,
```

Comment:
lgtm.

Reply:
Okay, lets go with the dynamic allocation approach.